showcase.json

```diff
@@ -2746,5 +2746,27 @@
     "twitter": "",
     "youtube": "",
     "submittableId": ""
-  }
+  },
+{
+  "name": "IC based storage service",
```

Comment:
Would you be open to using the product name "DCM Swiss" instead?

Reply:
Hi.
In my opinion, "IC based storage service" is a name of the project we developed. It is an abstract storage service to store and organize the user data (files and directories) in the IC ecosystem.
I suppose, the short name ics2 mght be good.

DCM Swiss is a name of the company. We will use this solution in several applications. I am sure, the current version of the ics2 will be extended later (I mentioned some notes in the readme file https://github.com/sergeybykov85/ic-storage-service/blob/main/ic-ss/README.md) and other tools will be built upon the ics2.

I guess, better to use  tthe name like ics2 as a name of the project. And yes, it will be utilized (it has been alrady started) by DCM Swiss

Reply:
Understood! Thank you for the clarification between the company name and product name!